### PR TITLE
Added the version 2.10.2 changelog

### PR DESCRIPTION
**Pull request by akuzm**
Updated the release notes with the 2.10.2 changelog

### CODE REVIEW

File: about/release-notes/index.md [hunk 1]
```diff
@@ -17,6 +17,37 @@ GitHub and be notified by email whenever a new release is available. On the
 click `Watch`, select `Custom` and then check `Releases`.
 </Highlight>
 
+
+## 2.10.2 (2023-04-20)
+
+**Bugfixes**
```
Comment (Loquacity) [inline about/release-notes/index.md:23]:
```suggestion
**Bug fixes**
```

**Comment by Loquacity** [comment]
Please make sure you're using the template at the bottom of the page when you write release notes.

**Comment by Loquacity** [comment]
@akuzm Ready to merge?

**Comment by akuzm** [comment]
> @akuzm Ready to merge? The site build failures are just for the test site, and there's nothing wrong with the actual build, FYI.

Yep, let's merge. It says it needs one approving review.

**Pull request merged**